r/R/shiny.R

```diff
@@ -177,7 +177,11 @@ parse_mmss <- function(x = "") {
 }
 
 require_shiny <- function(reason) {
-  if (!requireNamespace("shiny", quietly = TRUE)) {
+  if (!has_shiny()) {
     stop(reason, " requires shiny: install.packages('shiny')", call. = FALSE)
   }
 }
+
+has_shiny <- function() {
+  requireNamespace("shiny", quietly = TRUE)
+}
```

Comment:
```suggestion
}

```

Reply:
Thanks! I'm bringing in `air` in another PR (that I'll push soon)

Reply:
Any customization on air? Or: 

```r
usethis::use_air()

usethis::use_github_action(url = "https://github.com/posit-dev/setup-air/blob/main/examples/format-suggest.yaml")
```